code/nuxeo-apidoc-jsf-webengine/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,11 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 1
+Bundle-Name: nuxeo apidoc jsf webengine
+Bundle-SymbolicName: org.nuxeo.apidoc.jsf.webengine;singleton:=true
+Bundle-Version: 0.0.1
+Bundle-Vendor: Nuxeo
+Nuxeo-Require: org.nuxeo.apidoc.core,
+ org.nuxeo.apidoc.webengine
+ org.nuxeo.apidoc.jsf
+Fragment-Host: org.nuxeo.apidoc.webengine
+ 
```

Comment:
This is strange 🤔
![image](https://user-images.githubusercontent.com/3491744/79984075-f2bbfd80-84a8-11ea-9bb7-4247d94fcdca.png)
GitHub says there's no newline.

Reply:
i don't follow